pkg/services/playlist/playlistimpl/store.go

```diff
@@ -18,209 +14,3 @@ type store interface {
 	List(context.Context, *playlist.GetPlaylistsQuery) (playlist.Playlists, error)
 	Update(context.Context, *playlist.UpdatePlaylistCommand) (*playlist.PlaylistDTO, error)
 }
-
-type sqlStore struct {
-	db db.DB
-}
-
-func (s *sqlStore) Insert(ctx context.Context, cmd *playlist.CreatePlaylistCommand) (*playlist.Playlist, error) {
-	p := playlist.Playlist{}
-	err := s.db.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
-		uid, err := generateAndValidateNewPlaylistUid(sess, cmd.OrgId)
-		if err != nil {
-			return err
-		}
-
-		p = playlist.Playlist{
-			Name:     cmd.Name,
-			Interval: cmd.Interval,
-			OrgId:    cmd.OrgId,
-			UID:      uid,
-		}
-
-		_, err = sess.Insert(&p)
-		if err != nil {
-			return err
-		}
-
-		playlistItems := make([]playlist.PlaylistItem, 0)
-		for _, item := range cmd.Items {
-			playlistItems = append(playlistItems, playlist.PlaylistItem{
-				PlaylistId: p.Id,
-				Type:       item.Type,
-				Value:      item.Value,
-				Order:      item.Order,
-				Title:      item.Title,
-			})
-		}
-
-		_, err = sess.Insert(&playlistItems)
-
-		return err
-	})
-	return &p, err
-}
-
-func (s *sqlStore) Update(ctx context.Context, cmd *playlist.UpdatePlaylistCommand) (*playlist.PlaylistDTO, error) {
-	dto := playlist.PlaylistDTO{}
-	err := s.db.WithTransactionalDbSession(ctx, func(sess *sqlstore.DBSession) error {
-		p := playlist.Playlist{
-			UID:      cmd.UID,
-			OrgId:    cmd.OrgId,
-			Name:     cmd.Name,
-			Interval: cmd.Interval,
-		}
-
-		existingPlaylist := playlist.Playlist{UID: cmd.UID, OrgId: cmd.OrgId}
-		_, err := sess.Get(&existingPlaylist)
-		if err != nil {
-			return err
-		}
-		p.Id = existingPlaylist.Id
-
-		dto = playlist.PlaylistDTO{
-
-			Id:       p.Id,
-			UID:      p.UID,
-			OrgId:    p.OrgId,
-			Name:     p.Name,
-			Interval: p.Interval,
-		}
-
-		_, err = sess.Where("id=?", p.Id).Cols("name", "interval").Update(&p)
-		if err != nil {
-			return err
-		}
-
-		rawSQL := "DELETE FROM playlist_item WHERE playlist_id = ?"
-		_, err = sess.Exec(rawSQL, p.Id)
-
-		if err != nil {
-			return err
-		}
-
-		playlistItems := make([]models.PlaylistItem, 0)
-
-		for index, item := range cmd.Items {
-			playlistItems = append(playlistItems, models.PlaylistItem{
-				PlaylistId: p.Id,
-				Type:       item.Type,
-				Value:      item.Value,
-				Order:      index + 1,
```

Comment:
Here, and in xorm store, there is a `+ 1`. But not in the sqlx store inside Update.

Reply:
(Make sure  to `diff --color sqlx_store.go xorm_store.g`)